### ISSUE TITLE
Adding libslirp support from upstream

### ISSUE THREAD

**Issue by MojoJojoDojo** [rafaelvcaetano/melonDS-android#21]
I think that feature will be nice as it would allow an online connection for Android users for the first time.

**Comment by MojoJojoDojo** [comment]
Wrong place oops

**Comment by MojoJojoDojo** [comment]
6

**Comment by rafaelvcaetano** [comment]
Yeah, I'm following the development on that end. I'm just waiting for the next PC release to start working on it to make sure I don't deal with incomplete stuff that might cause problems later.

**Comment by HJMBgit84** [comment]
That would be awesome....! :+1:

**Comment by MojoJojoDojo** [comment]
Well I compiled it myself and it worked pretty good on Windows as far as I know but I guess I will wait for further developments.

**Comment by imdaveho** [comment]
I'm curious about this as well. 0.9 just released on the PC version. 

Curious--because I don't have the historical context. I know that the PC version has mentioned libslirp, but has it already been implemented, so that porting it over would be easier? Or were you waiting for the next version to have all the fixes so that you'll be working off a more complete base to integrate it on this version? Thanks! @rafaelvcaetano

**Comment by shabbirmahmud** [comment]
Duplicate of #38
